code/game/gamemodes/wizard/godhand.dm

```diff
@@ -78,6 +78,7 @@
 	on_use_sound = 'sound/magic/disintegrate.ogg'
 	icon_state = "disintegrate"
 	item_state = "disintegrate"
+	needs_permit = 0
```

Comment:
```suggestion
	needs_permit = FALSE
```
Tiny nitpick. If something is a boolean, we use `TRUE` for `1` and `FALSE` for `0`. They both compute to the numericals ingame, its just for readability.

Reply:
This opens a can of worms in my mind - everywhere else I'd looked was using 0 and 1. I'd offer to go back and fix all those as part of this PR, except that I accidentally pulled from my master branch, and I don't want to dirty that any further.